src/supported.jl

```diff
@@ -65,11 +65,17 @@ function supported_objective(primal_model::MOI.ModelLike) where T
     if !supported_obj(obj_func_type)
         error("Objective functions of type ", obj_func_type," are not implemented")
     end
-    return 
+
+    obj_func = MOI.get(primal_model, MOI.ObjectiveFunction{obj_func_type}())
+    if length(filter(t -> t.coefficient != zero(Float64), obj_func.terms)) == 0
```

Comment:
```Julia
if all(c -> iszero(c.coefficient), obj_func.terms)
```

Reply:
Oh, nice! While we're at it, wouldn't `any` be better? It could short-circuit its execution more often, I think.